cardano-api-test/test/Test/Cardano/Api/Typed/Script.hs

```diff
@@ -126,6 +128,13 @@ prop_roundtrip_ScriptData =
     sData <- H.forAll genScriptData
     sData === fromAlonzoData (toAlonzoData sData)
 
+prop_roundtrip_ScriptDataHash :: Property
+prop_roundtrip_ScriptDataHash =
+  H.property $ do
+    sData <- H.forAll genScriptData
+    hashScriptData sData === ScriptDataHash (Alonzo.hashData (toAlonzoData sData :: Alonzo.Data StandardAlonzo))
```

Comment:
What is the purpose of this test? The definition of `hashScriptData` is exactly what you have there: `ScriptDataHash . Alonzo.hashData . toAlonzoData`.

Reply:
I was debugging why the ledger was saying I was missing a datum hash. I can remove it.